sys/Makefile.dep

```diff
@@ -13,7 +13,11 @@ ifneq (,$(filter arduino,$(USEMODULE)))
   FEATURES_OPTIONAL += arduino_pwm
   FEATURES_OPTIONAL += periph_adc
   FEATURES_REQUIRED += periph_gpio
-  USEMODULE += xtimer
+  USEMODULE += ztimer_usec
+  USEMODULE += ztimer_msec
+  ifneq (,$(filter periph_rtt,$(USEMODULE)))
+    USEMODULE += ztimer_msec_periph_rtt
+  endif
```

Comment:
```suggestion
  USEMODULE += ztimer_usec
  USEMODULE += ztimer_msec
```

Reply:
The later will be addressed with https://github.com/RIOT-OS/RIOT/pull/16553

Reply:
Thanks, exactly what I needed !